azure-function-app/massdriver.yaml

```diff
@@ -142,7 +143,6 @@ params:
       description: Application configuration
       type: object
       required:
-        - cidr
```

Comment:
I can't comment on it, but you'll want to remove `cidr` on line `151` if you're going to replace it w/ `netowork` 👍

Reply:
Not sure how I missed that, but thank you! It was removed.